llvm/lib/IR/RuntimeLibcalls.cpp

```diff
@@ -277,5 +277,10 @@ void RuntimeLibcallsInfo::initLibcalls(const Triple &TT) {
       setLibcallName(RTLIB::MULO_I64, nullptr);
     }
     setLibcallName(RTLIB::MULO_I128, nullptr);
+  } else {
```

Comment:
Should this be `if (TT.isOSEmscripten())` ?

Reply:
I suspect this predates when we had an emscripten triple, and it should probably be behind `isOSEmscripten()`

Reply:
Not looking to change behavior here, this should be done separately